charts/everest/charts/common/templates/_create_default_psp.yaml.tpl

```diff
@@ -0,0 +1,91 @@
+# Create default pod scheduling policies.
+#
+{{- define "everest.createDefaultPsp" }}
+apiVersion: everest.percona.com/v1alpha1
+kind: PodSchedulingPolicy
+metadata:
+  name: everest-default-pxc
+  finalizers:
+    - everest.percona.com/readonly-protection
+  annotations:
+      "helm.sh/hook": post-install,pre-upgrade
+      "helm.sh/resource-policy": keep
```

Comment:
Why do we need this `"helm.sh/resource-policy": keep` annotation?

Reply:
What I meant to ask was - why should these PSP never be deleted by Helm in the future? 🤔

Reply:
ah, I ment this is protection from being removed by hook controller. 
We remove these PSPs in the `delete-hook` manually